<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home_page</class>
 <widget class="QMainWindow" name="home_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ristorante Emilia</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonToLogin">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Dipendente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonToHomeCliente">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Cliente</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
